collection/eliane.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "eliane",
+    "meaning": "Feminine france name. The meaning of the name Eliane is: Jehovah is God or in France: Daughter of the Sun.",
+    "translations": {
+        "heb": "אֶלִיעַנָה", 
+        "por": "eliane",
+        "eng": "eliana",
```

Comment:
Can you please remove the comma after the last element in the translations object?